t2f/app-elements/pdf-export/ta-print.html

```
@@ -193,6 +195,8 @@
                     return [
                       {text: printDate(dsa.start_date), alignment: 'left'},
                       {text: printDate(dsa.end_Date), alignment: 'center'},
+                      {text: print(dsa.dsa_region_name.split(' - ')[0]), alignment: 'center'},
```

Please make this in a function and check the string structure and existence of( - ), otherwise it will fail silently and not generate the PDF ( and create a lot of headache in the future )